src/eve/visitors.py

```diff
@@ -102,6 +107,8 @@ def apply(cls, tree, init_var, foo, bar=5, **kwargs):
 
     """
 
+    previsitors: ClassVar[Optional[Tuple[PrevisitCallable]]] = None
```

Comment:
The current typing is incorrect: it represents a tuple with only one `PrevisitCallable` item. To indicate an undefined number of elements in the tuple, the notation is:
 ```suggestion
    previsitors: ClassVar[Optional[Tuple[PrevisitCallable, ...]]] = None
```
(Note that this is different for Tuples and Lists, since `List[PrevisitCallable]` is actually correct)

Reply:
Yep, looks right. Nice catch.